docs/src/pages/guides/examples.md

```diff
@@ -186,7 +186,7 @@ probes:
   - id: '1'
     name: Probing Github
```

Comment:
oh ya, can you please change the name from "Probing Github": since it is not probing github... (looks like from old copy paste)